src/style.css

```diff
@@ -1 +1,96 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-weight: 500px;
+}
+
+@font-face {
+  font-family: "Roboto-bold";
+  src: url(./fonts/Roboto-Bold.ttf);
+  font-weight: 800;
+}
+
+.header {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  justify-content: space-between;
+  min-width: 1024px;
+  max-width: 1200px;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header:hover {
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  margin: 10px 0 10px 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.nav__list {
+  display: flex;
+  flex-wrap: wrap;
+
+  align-items: center;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.nav__list:last-child {
+  margin-right: 30px;
+}
+
+.nav__list:first-child {
+  margin-left: 20px;
+}
+
+.nav__item {
+  margin-right: 20px;
```

Comment:
Please remove margin-right for last item
![image](https://user-images.githubusercontent.com/54936044/231001643-9ac236fe-f36f-48ee-8d2c-bfee8cc0cc93.png)